app/models/proposal/exporter.rb

```diff
@@ -15,4 +27,25 @@ def json_values(proposal)
         description: strip_tags(proposal.description)
       }
     end
-end
+    
+    def headers
+      [
+        I18n.t("admin.proposals.index.list.id"),
+        I18n.t("admin.proposals.index.list.title"),
+        I18n.t("admin.proposals.index.list.author"),
+        I18n.t("admin.proposals.index.list.summary"),
+        I18n.t("admin.proposals.index.list.description"),
+      ]
+    end
+
+    def csv_values(proposal)
+      [
+        proposal.id.to_s,
+        proposal.title,
+        proposal.author.email,
+        proposal.summary,
+        proposal.description
```

Comment:
What do you think about removing the description field from the CSV altogether?

In some cases, it can be quite long (up to 6000 characters), and I’m not sure if it adds much value for this kind of export.